src/router/index.tsx

```diff
@@ -3,14 +3,13 @@ import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Main from '../components/main';
 import DatasetsPage from '../components/pages/datasets-page';
 import { routes } from './routes';
-import EditDatasetPage from "../components/pages/edit-dataset-page";
 
 const Router: FC = () => (
   <BrowserRouter>
     <Routes>
       <Route path={routes.home} element={<Main />}>
         <Route path={routes.catalogId} element={<DatasetsPage />} />
-        <Route path={routes.datasetId} element={<EditDatasetPage />} />
+        <Route path={routes.datasetId} element={<h1>single dataset</h1>} />
```

Comment:
why?

Reply:
Hukser ikke å ha brukt/laget `edit-datasets-page`. Kan hende jeg lagde den som en midlertidig ting, eller kanskje du eller Terje har lagdet den?

Reply:
Vi lagde den for at terje skulle kunne jobbe der, ja. Hvis du mener at den skal slettes, ok greit, men ta det i en egen pr. Det er utenfor scope på denne pr'en å slette diverse som du ikke jobber med

Reply:
Skjønner. Ok, da fikser jeg det nå.